Src/IronPython/Hosting/PythonCommandLine.cs

```diff
@@ -291,12 +291,10 @@ public sealed class PythonCommandLine : CommandLine {
         private void InitializeExtensionDLLs() {
             string dir = Path.Combine(PythonContext.InitialPrefix, "DLLs");
             if (Directory.Exists(dir)) {
-                foreach (string file in Directory.EnumerateFiles(dir)) {
-                    if (file.ToLower().EndsWith(".dll")) {
-                        try {
-                            ClrModule.AddReference(PythonContext.SharedContext, new FileInfo(file).Name);
-                        } catch {
-                        }
+                foreach (string file in Directory.EnumerateFiles(dir, "*.dll")) {
```

Comment:
Note that with this change we might also match non-DLL files. The wildcard is actually equivalent to \*.dll\* so you could end up matching other stuff (e.g. a.dllb).